config/locales/en.txt

```diff
@@ -3261,6 +3270,9 @@
   send_feature_email_success: Delivered feature mail.
   send_feature_email_denied: Only the admin can send feature mail.
 
+  # project general
+  project_nil_date_str: "[:NONE]"
```

Comment:
@mo-nathan: Is there a better place in this file to put this?

Reply:
Seems ok.  I'd probably just use `:NONE.t` directly instead of creating a translation specifically for this.

Reply:
I'll use :NONE directly. 
It's easy enough to use a translation in the future if needed.

Reply:
Incidentally I would love to do a purge of our duplicate translation strings. The proliferation seems unmaintainable, doesn't it? Particularly for translators.

Reply:
The whole point of the `[:tag]` syntax was precisely to keep it maintainable.  This way there is no cost to duplicating strings, and that's what I wanted to encourage.  Who's to say that in Inuit or Cherokee they would use the same phrase as we do in English for this purpose?

Reply:
Ohhhhhh. I didn't understand this!

Reply:
Thanks, @mo-nathan.  Certainly.  I reuse these simple one-off words all the time, too.

FWIW, I was responding to and pushing back against a perceived desire to simplify more than necessary, and against the worry that duplicates incur maintenance debt.  I felt that needed to be clarified, and share why I designed it the way I did.

In short, I guess, I advocate leaving it the way it is.  I don't think it gains us anything to put in even a few hours' work to remove all of the "trivial" duplicates (the strings that are literally just synonyms of other terms with nothing added or modified).  They do no harm.  It does cost us developers or admins, and they don't even cost translators anything.

Reply:
Totally fine with that.  Doesn't seem like a rabbit hole worth going down at the moment.